ci/scripts/run_pytest.sh

```diff
@@ -23,14 +23,30 @@ export BIONEMO_DATA_SOURCE="${BIONEMO_DATA_SOURCE:-pbss}"
 export PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True
 source "$(dirname "$0")/utils.sh"
 
+# Parse options
+# Check for the --skip-docs flag
+SKIP_DOCS=false
+for arg in "$@"; do
+    if [ "$arg" == "--skip-docs" ]; then
+        SKIP_DOCS=true
+        break
+    fi
+done
+
 if ! set_bionemo_home; then
     exit 1
 fi
 
 python -m coverage erase
 
 error=false
+
+# Loop through directories and skip docs/ if the flag is set
 for dir in docs/ ./sub-packages/bionemo-*/; do
+    if [ "$SKIP_DOCS" == true ] && [ "$dir" == "docs/" ]; then
+        echo "Skipping pytest in $dir because --skip-docs flag is set"
+        continue
+    fi
     echo "Running pytest in $dir"
     python -m coverage run --parallel-mode --source=bionemo \
     -m pytest -v --nbval-lax --durations=0 --durations-min=60.0 "$dir" || error=true
```

Comment:
This will still run the `examples` long notebook tests that are within regular packages due to the `--nbval-lax` argument. Is the intention not to run notebooks? If so maybe make that the argument and if so skip both docs and drop `--nbval-lax`.